static/js/chartHandler.js

```diff
@@ -1,45 +1,175 @@
-let $voltageChart = $("#voltage-Chart");
-let compute = 'force' //Change later :) 
-var newChart;
+//Grabbing chart from 
+let $chart_element = $("#chart-element");
+var new_chart;
+let previousX;
+let previousY;
 
+
+//Downloads a png, only appears after you hit graph. 
+//Creates anchor tag -> converts the chart to a png
+//Forces click, and a download should happen
+const chartDownload = () => {
+  let a = document.createElement('a');
+  a.href = new_chart.toBase64Image();
+  a.download = 'chart.png';
+  a.click();
+};
+
+//This is the drop downs for x-value and y-value to be graphed
+const createDropDown = () => {
+  const urlParams = new URLSearchParams(window.location.search);
+  let select_X = document.getElementById('x-values-input');
+  let select_Y = document.getElementById('y-values-input');
+  let inputs = [
+    'Voltage',
+    'Length',
+    SolenoidParameters.R0,
+    SolenoidParameters.RA,
+    SolenoidParameters.X,
+    'Force',
```

Comment:
I understand that we can't have Voltage be SolenoidParameters.VOLTAGE because it would be 'v' but something about this inconsistency doesn't sit super well with me